eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionWriter.java

```diff
@@ -15,10 +15,24 @@
  */
 package uk.ac.ebi.eva.accession.pipeline.io;
 
-import org.springframework.batch.item.support.CompositeItemWriter;
+import org.springframework.batch.item.ItemWriter;
 
-import uk.ac.ebi.eva.accession.core.persistence.SubmittedVariantEntity;
+import uk.ac.ebi.eva.accession.core.ISubmittedVariant;
+import uk.ac.ebi.eva.accession.core.SubmittedVariantAccessioningService;
 
-public class SummaryFileWriter extends CompositeItemWriter<SubmittedVariantEntity> {
-    // TODO: implement
+import java.util.List;
+import java.util.Map;
+
+public class AccessionWriter implements ItemWriter<ISubmittedVariant> {
+
+    private SubmittedVariantAccessioningService service;
+
+    public AccessionWriter(SubmittedVariantAccessioningService service) {
+        this.service = service;
+    }
+
+    @Override
+    public void write(List<? extends ISubmittedVariant> variants) throws Exception {
+        Map<Long, ISubmittedVariant> accessions = service.getOrCreateAccessions(variants);
```

Comment:
For auditing purposes, it was agreed that the database entities would have a creation date. Can you please modify it the class accordingly?

Also, shall we change `SubmittedVariantEntity` it to be a `@Document` instead of an `@Entity`, or shall we wait until the benchmarks are finished? If we decide to wait we have to leave the JIRA ticket open.

Reply:
I'm adding the the date for testing purposes until it's done in accession-commons, but we won't add the mongo part in this PR as agreed.